pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -902,9 +907,17 @@ type mountDevice string
 // TODO: Also return number of mounts allowed?
 func (self *awsInstanceType) getEBSMountDevices() []mountDevice {
 	// See: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html
+	// We will generate "ba", "bb", "bc"..."bz", "ca", ..., up to DefaultMaxEBSVolumes
 	devices := []mountDevice{}
-	for c := 'f'; c <= 'p'; c++ {
-		devices = append(devices, mountDevice(fmt.Sprintf("%c", c)))
+	for i, first, second := 0, 'b', 'a'; i < DefaultMaxEBSVolumes; i++ {
```

Comment:
This might be clearer written as two loops (over first, then over second, with a separate check for # of volumes.  But it's up to you!